recipes-misc/testnet/testnet_2.0.0.bb

```diff
@@ -0,0 +1,37 @@
+DESCRIPTION = "Testnet - test network for firewall blocks on Izuma Edge"
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/git/pe-utils/LICENSE;md5=1dece7821bf3fd70fe1309eaa37d52a2"
+
+SRC_URI="\
+git://git@github.com/PelionIoT/pe-utils.git;protocol=https;name=pe-utils;destsuffix=git/pe-utils \
+"
+SRCREV_pe-utils = "2.0.11"
+
+inherit pkgconfig gitpkgv edge
+
+PV = "1.0+git${SRCPV}"
+PKGV = "1.0+git${GITPKGV}"
+PR = "r0"
+
+DEPENDS = ""
+RDEPENDS_${PN} += "busybox openssl"
+
+RM_WORK_EXCLUDE += "${PN}"
+
+FILES_${PN} = "\
+${EDGE_BIN}/ \
+/etc/tmpfiles.d/userdata-tmpfiles.conf \
```

Comment:
```suggestion

```

Reply:
testnet needs to be added to https://github.com/PelionIoT/meta-pelion-edge/blob/testnet/recipes-core/images/console-image.bb to be included in the image.

Reply:
Yep, I have done those already -I just forgot them from this PR last night. They are now in.
We have actually TWO of those files, why two?

Reply:
`meta-lmp-support/recipes-core/images/console-image-lmp.bb` and `recipes-core/images/console-image.bb` - which one do we really use? Or do we somehow use both?

Reply:
I suspect that the one in recipes-core is for a non-LmP Yocto build, but I doubt we support that anymore.

Reply:
Okay, that would definitely explain it.